src/element/collision.ts

```diff
@@ -78,6 +93,42 @@ export function hitTest(
       leftY
     ] = getDiamondPoints(element);
 
+    if (element.backgroundColor !== "transparent") {
+      // TODO: remove this when we normalize coordinates globally
+      if (topY > bottomY) [bottomY, topY] = [topY, bottomY];
+      if (rightX < leftX) [leftX, rightX] = [rightX, leftX];
```

Comment:
This hack is necessary when we flip the shape (via resize) horizontally/vertically.

Reply:
Perhaps #342 helps with this

Reply:
Oh, cool, didn't notice your PR. Yes, it'll likely fix that. So maybe wait for your PR to land before merging this?